src/pma.h

```diff
@@ -480,6 +502,18 @@ class pma_entry final {
         }
         return address >= get_start() && get_length() >= length && address - get_start() <= get_length() - length;
     }
+
+    /// \brief  Writes data to pma memory
+    /// \param paddr Destination address within pma range
+    /// \param data Source data
+    /// \param size Data size
+    void write_memory(uint64_t paddr, const unsigned char *data, uint64_t size);
+
+    /// \brief  Sets pma memory to a given value
+    /// \param paddr Destination address within pma range
+    /// \param value Value to write
+    /// \param size Data size
+    void set_memory(uint64_t paddr, unsigned char value, uint64_t size);
```

Comment:
Why not simply use `write_memory` passing the address of your word? Is there some restriction on this use?